DesignSafe-CI/portal
# Task/DES-1465 - Update Manage Experiments Modal

## Overview: ##
This was originally intended to remove the reliance on a .json file to properly display certain fields from Experimental metadata, but the risks involved are not worth the benefits at this time. Instead we've updated the modal for the new requirements in the Amends/Revisions pipelines.

- Refactor template to match more current design
- Remove old author sorting (this will be replaced as part of a separate task)
- Allow users to modify Experimental Facility, Equipment, and Type once an Experiment is created
- Prevent users from deleting Experiments which have a DOI (TODO)

## PR Status: ##


## Related Jira tickets: ##

* [DES-1465](https://jira.tacc.utexas.edu/browse/DES-1465)

## Summary of Changes: ##
- Refactor template to match more current design
- Remove old author sorting (this will be replaced as part of a separate task)
- Allow users to modify Experimental Facility, Equipment, and Type once an Experiment is created
- Prevent users from deleting Experiments which have a DOI (TODO)

## Testing Steps: ##
1. Open an Experimental Project
2. Create an Experiment with any details
3. You should be able to Edit that Experiment and change the Facility, Equipment, and Type fields
4. A notification should appear preventing users from deleting Experiments with a DOI
